src/drive/web/modules/drive/Toolbar/components/ScanItem.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react'
+import { Icon, translate } from 'cozy-ui/transpiled/react'
+
+import Scanner from './Scanner'
+import toolbarContainer from '../toolbar'
+import PortaledQueue from './PortaledQueue'
+
+import {
+  startMediaBackup,
+  cancelMediaBackup
+} from 'drive/mobile/modules/mediaBackup/duck'
+
+import { connect } from 'react-redux'
+/**
+ *
+ */
+
+const ScanItemMenu = translate()(({ status, onClick, t, online }) => {
+  const offlineMessage = () => {
+    return alert(t('Scan.error.offline'))
+  }
+  const uploadingMessage = () => {
+    return alert(t('Scan.error.uploading'))
+  }
+  const actionOnClick = (() => {
+    if (status === 'uploading') return uploadingMessage
+    if (!online) return offlineMessage
+    return onClick
+  })()
+
+  return (
+    <span className="u-pl-1 u-flex u-pt-half u-pb-half" onClick={actionOnClick}>
+      <Icon icon="camera" />
+      <span className="u-pl-half">{t('Scan.scan_a_doc')}</span>
+    </span>
+  )
+})
+class ScanItem extends Component {
+  render() {
+    const { displayedFolder, stopMediaBackup, startMediaBackup } = this.props
+    return (
+      <>
+        <Scanner
+          dirId={displayedFolder.id} //Pour savoir où uploader
+          pluginConfig={{
+            sourceType: 1 // Camera
+          }}
+          generateName={() => {
+            const date = new Date()
+            return `Scan_${date.toISOString()}.jpg`
+          }}
+          onConflict={'rename'}
+          onBeforeUpload={() => stopMediaBackup()}
+          onFinish={() => startMediaBackup()}
+        >
+          {({ status, error, onClick, filename, onClear, online }) => {
```

Comment:
Why does `online` come from the Scanner ?

Reply:
And maybe `onClick` could be renamed to something like `startScanner`?

Reply:
I though it's easier that the Scanner handles by itself the online / offline status. In a near future, Scanner will be able to write in localStorage the content of the scan if offline and then sync it later. 

Agreed with `startScanner` thanks for the reminder :)